src/main/cpp/RLBotInterface/RLBotInterface/Interface.cpp

```diff
@@ -0,0 +1,164 @@
+#include "Interface.hpp"
+
+#include <atomic>
+#include <Messages.hpp>
+
+#include "CallbackProcessor\CallbackProcessor.hpp"
+
+namespace Interface
+{
+	static std::atomic_bool bInitialized = false;
+
+	extern "C" bool RLBOT_CORE_API IsInitialized()
+	{
+		return bInitialized;
+	}
+
+	void Callback(unsigned int id, RLBotCoreStatus status)
+	{
+		DEBUG_LOG("SendChat callback has been called! ID: %i - Status: %i\n", id, status);
+	}
+
+	DWORD WINAPI Initialize(void*)
+	{
+#ifdef _DEBUG
+		AllocConsole();
+		AttachConsole(GetCurrentProcessId());
+		freopen("CONOUT$", "w", stdout);
+#endif
+
+		DEBUG_LOG("====================================================================\n");
+		DEBUG_LOG("RLBot Core Interface\n");
+		DEBUG_LOG("====================================================================\n");
+		DEBUG_LOG("Initializing...\n");
+
+		if (!FileMappings::Initialize())
+			return ERROR_FUNCTION_FAILED;
+
+		if (!CallbackProcessor::Initialize())
+			return ERROR_FUNCTION_FAILED;
+
+		bInitialized = true;
+		DEBUG_LOG("RLBot Core Interface has been successfully initialized!\n");
+
+		/*while (1)
```

Comment:
Clean up commented out test code.

Reply:
This is ccman's, so I'd rather not in this case.